### PR TITLE
Helper to wrap text into direction markers

### PR DESCRIPTION
**Pull request by imobachgs**
Related pull requests:

* https://github.com/yast/yast-packager/pull/348
* https://github.com/yast/yast-installation/pull/687

### CODE REVIEW

File: library/general/src/lib/ui/text_helpers.rb [hunk 1]
```diff
@@ -53,5 +53,17 @@ def wrap_text(text, wrap = 76, separator: " ", prepend_text: "",
...
       lines.join("\n")
     end
+
+    # Wrap a given text in direction markers
+    #
+    # @param [String] text to be wrapped
+    # @param [String] language code (it gets the current one by default)
+    # @return [String] wrapped text
+    def direct_richtext(text, lang = Yast::Language.language)
+      Yast.import "Language"
```
Comment (mvidner) [inline library/general/src/lib/ui/text_helpers.rb]: I think this is too late for the parameter declaration.
This method is at an intersection of Language and Text but with more of the language, so maybe we should move it to a place that already import Language?

File: library/general/src/lib/ui/text_helpers.rb [hunk 2]
```diff
@@ -53,5 +53,17 @@ def wrap_text(text, wrap = 76, separator: " ", prepend_text: "",
 
       lines.join("\n")
     end
+
+    # Wrap a given text in direction markers
+    #
+    # @param [String] text to be wrapped
+    # @param [String] language code (it gets the current one by default)
+    # @return [String] wrapped text
```
Comment (mvidner) [inline library/general/src/lib/ui/text_helpers.rb]: We should say (and test) that *text* may contain tags which will be passed through (not escaped)

File: library/general/src/lib/ui/text_helpers.rb [hunk 3]
```diff
@@ -53,5 +53,17 @@ def wrap_text(text, wrap = 76, separator: " ", prepend_text: "",
...
+    def direct_richtext(text, lang = Yast::Language.language)
```
Comment (mvidner) [inline library/general/src/lib/ui/text_helpers.rb]: This name would confuse me at the call site.
What about `div_with_direction` ?

File: library/general/src/lib/ui/text_helpers.rb (hunk 1 above)
Comment (imobachgs) [reply library/general/src/lib/ui/text_helpers.rb]: Or we might just move the initialization to the line below.

```ruby
div_with_direction(text, lang = nil)
  Yast.import "Language"
  lang ||= Yast::Language.language
  #...
end
```

**Pull request merged**